src/components/Button.js

```diff
@@ -1,6 +1,19 @@
 import React from 'react';
 import PropTypes from 'prop-types';
 
-export const Button = ({ id }) => <button>{ id }</button>
+export const Button = ({ name, color, wide }) => {
+  const orange = '#f6913f';
+  const clearGrey = '#e0e0e0';
+  const styles = {
+    backgroundColor: color ? orange : clearGrey,
+    width: wide ? '50%' : '25%',
+    height: '100px',
+  }
+  return <button style={ styles } >{ name }</button>
+};
 
```

Comment:
Hi, let us use defaultProps to set the default (orange) see [requirement](https://gitlab.com/microverse/guides/projects/requirements_screenshots/raw/master/images/react_reducx/calculator/milestone3/colorProps.png). That way we can just assign color on `line 8` directly and it would take it either form the color props from `buttonPannel` or from the defaultProps if not set.